core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Accepts connections from remote clients so that they can be used as additional build agents}
+  ${%Accepts connections from remote clients so that they can be used as additional build agents.  This protocol is insecure.}
```

Comment:
Is this an action for me to take or a note to the maintainer who would merge this change?  If so, can you describe more about what needs to be done from my end?